fastn-core/src/auth/validator.rs

```diff
@@ -0,0 +1,56 @@
+pub fn validate_strong_password(password: &str) -> Result<(), validator::ValidationError> {
+    let mut has_uppercase = false;
+    let mut has_lowercase = false;
+    let mut has_number = false;
+    let mut has_special = false;
```

Comment:
Not fond of this kind of validation. "dexter likes to write email validators" is strong, and easier to remember password than say "$%ASsdfer$123".

Reply:
<img width="839" alt="image" src="https://github.com/fastn-stack/fastn/assets/58662/452de0c6-b638-4a1d-8e6e-71c3eaa9c925">

Maybe we should use paspio or https://github.com/dewan-ahmed/PassMeRust to check password strength instead.

Reply:
using [zxcvbn](https://docs.rs/crate/zxcvbn/2.2.2) https://github.com/fastn-stack/fastn/pull/1722/commits/7d00717b941c4b541756f7192ac54cc43a57e7ef